emails/register/html.ejs

```diff
@@ -1,9 +1,12 @@
 <p>Dear <%= name %>,
 
-<p>We are very excited that you have joined the Humanitarian ID community. As a first step, we need you to confirm your account by simply following the link below. Please make sure you confirm your account within the next 7 days, otherwise it will be deleted.</p>
-
-<p>As we expand integration with humanitarian websites, your account will gain you access to a wide range of resources. But, of course, it will also give you access to the Humanitarian ID contact management solution!</p>
+<p>Thank you for joining Humanitarian ID. To complete the registration, we need you to confirm your account by clicking on the link below. Please note that this is a one-time use link and it will remain valid for 24 hours.</p>
 
 <a href="<%= reset_url %>"><%= reset_url %></a>
 
+<p>In case the link has expired, you can request a new one, by clicking here: <a href="https://auth.humanitarian.id/password">Request new link</a></p>
```

Comment:
Period at the end?

```
<p>In case the link has expired, you can request a new one, by clicking here: <a href="https://auth.humanitarian.id/password">Request new link</a>.</p>
```